monitor/events/registry.py

```diff
@@ -33,6 +33,7 @@ class Registry:
     splash_load = Event[Callable[[bool], None]]('SPLASH_LOAD')
     system_reboot = Event[Callable[[], None]]('SYSTEM_REBOOT')
     system_shutdown = Event[Callable[[], None]]('SYSTEM_SHUTDOWN')
+    switch_mode = Event[Callable[[], None]]('MODE_SWITCH')
```

Comment:
make sure the name is the same is as the event name, in this case `MODE_SWITCH` should be called `mode_switch`